src/components/App.js

```diff
@@ -1,6 +1,8 @@
 import '../App.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+// eslint-disable-next-line no-unused-vars
```

Comment:
### [Optional]
It would be nice if you could solve the codebase instead of making use of Eslint inline comment.
If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇